ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/SearchForWebmap/SearchForWebmap.cpp

```diff
@@ -133,51 +133,17 @@ void SearchForWebmap::searchNext()
 void SearchForWebmap::loadSelectedWebmap(int index)
 {
   if (!m_webmaps)
-    return;
-
-  PortalItem* selectedItem = m_webmaps->at(index);
-  if (!selectedItem)
-    return;
-
-  if (m_selectedItem)
-    m_selectedItem->disconnect();
-
-  m_selectedItem = selectedItem;
-
-  connect(m_selectedItem, &PortalItem::loadStatusChanged, this, [this]
-  {
-    if (!m_selectedItem || m_selectedItem->loadStatus() != LoadStatus::Loaded)
-      return;
-
-    if (m_map)
-    {
-      delete m_map;
-      m_map = nullptr;
-    }
-
-    m_mapLoadeError.clear();
-    emit mapLoadErrorChanged();
-
-    m_map = new Map(m_selectedItem, this);
-
-    connect(m_map, &Map::errorOccurred, this, [this]()
```

Comment:
do we still want to have this in the sample to catch map errors?

Reply:
Good point. I will put that back in.